card/lib/card/format.rb

```diff
@@ -8,20 +8,18 @@ class Format
     INCLUSION_MODES  = { :closed=>:closed, :closed_content=>:closed, :edit=>:edit,
       :layout=>:layout, :new=>:edit, :setup=>:edit, :normal=>:normal, :template=>:template } #should be set in views
     
-    cattr_accessor :ajax_call, :registered, :max_depth
+    cattr_accessor :ajax_call, :registered
     [ :perms, :denial_views, :closed_views, :error_codes, :view_tags, :aliases ].each do |acc|
       cattr_accessor acc
       self.send "#{acc}=", {}
     end
-    # TODO: these should come from Cardio.config
-    @@max_char_count = 200
-    @@max_depth      = 20
```

Comment:
I think next I'm going to do these changes to move a few settings like this to config.